### ISSUE TITLE
Cleanup code indentation

### ISSUE THREAD

**Issue by magnus-madsen** [flix/book#20]
E.g.

![image](https://user-images.githubusercontent.com/2287953/178151181-43a6d2f8-0dac-40ab-8435-408a91c27926.png)

**Comment by jaschdoc** [comment]
Yeah this is from chapter 14 / tips and tricks which wasn't ready for merging :)
